src/main/java/io/seqera/tower/cli/responses/TaskView.java

```diff
@@ -0,0 +1,150 @@
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.responses;
+
+import io.seqera.tower.cli.utils.TableList;
+
+import java.io.PrintWriter;
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TaskView extends Response {
+
+    public final Map<String, Object> general;
+    public final String command;
+    public final String environment;
+    public final Map<String, Object> times;
+    public final Map<String, Object> resources;
+    public final Map<String, Object> usage;
+
+    public TaskView(
+            Map<String, Object> general,
+            String command,
+            String environment,
+            Map<String, Object> times,
+            Map<String, Object> resources,
+            Map<String, Object> usage
+    ) {
+        this.general = general;
+        this.command = command;
+        this.environment = environment;
+        this.times = times;
+        this.resources = resources;
+        this.usage = usage;
+    }
+
+    @Override
+    public Object getJSON() {
+        Map<String, Object> data = new HashMap<>();
+
+        data.put("general", general);
+
+        if (command != null) {
+            data.put("command", command);
+        }
+
+        if (environment != null) {
+            data.put("environment", environment);
+        }
+
+        if (!times.isEmpty()) {
+            data.put("times", times);
+        }
+
+        if (!resources.isEmpty()) {
+            data.put("resources", resources);
+        }
+
+        if (!usage.isEmpty()) {
+            data.put("usage", usage);
+        }
+
+        return data;
+    }
+
+    @Override
+    public void toString(PrintWriter out) {
+        out.println(ansi(String.format("%n%n    @|bold  %s|@", "General")));
+        TableList tableGeneral = new TableList(out, 2);
+        tableGeneral.setPrefix("    ");
+        tableGeneral.addRow("Task ID", general.get("taskId").toString());
+        tableGeneral.addRow("ID", general.get("id").toString());
+        tableGeneral.addRow("Working Directory", general.get("workDir").toString());
+        tableGeneral.addRow("Status", general.get("status").toString());
+        tableGeneral.print();
+
+        if (command != null) {
+            out.println(ansi(String.format("%n    @|bold Command|@")));
+            out.println(ansi("    -------"));
+            out.println(ansi(String.format("    %s", command.replaceAll("\n", "\n    "))));
+        }
+
+        if (environment != null) {
+            out.println(ansi(String.format("%n    @|bold Environment|@")));
+            out.println(ansi("    -----------"));
+            out.println(ansi(String.format("    %s", environment.replaceAll("\n", "\n    "))));
+        }
+
+        if (!times.isEmpty()) {
+            out.println(ansi(String.format("%n%n    @|bold  %s|@", "Execution time")));
```

Comment:
Add a third column with the description, like in the website:
![Screenshot from 2021-11-19 07-05-28](https://user-images.githubusercontent.com/1315429/142573584-bf572127-1e4b-4d8b-bdc7-f7f851b4a56e.png)